Dockerfile.moth-devel

```diff
@@ -16,6 +16,5 @@ COPY devel /app/
 COPY example-puzzles /puzzles/
 COPY theme /theme/
 
-WORKDIR /moth/
```

Comment:
Is this required for the Python3.7 fix?

Reply:
No, but this path doesn't exist on the devel server, and nothing will be written here while it's running. It just confuses path-resolution in the HTTP server more